io.github.pemsley.coot.yaml

```diff
@@ -0,0 +1,511 @@
+      - -Dman-pages=false
+      - -Dmedia-gstreamer=disabled
+      - -Dvulkan=disabled
+    build-options:
+      environment:
+        DESTDIR: "/app"
+        CPPFLAGS: "-DG_DISABLE_ASSERT -DG_DISABLE_CAST_CHECKS"
+    build:
+      commands:
+        - meson setup build
+        - meson compile -C build
+        - meson install -C build
+    post-install:
+      commands:
+        - glib-compile-schemas /app/share/glib-2.0/schemas
+        - gtk4-update-icon-cache -f -t /app/share/icons/hicolor
+        - gio-querymodules /app/lib/gtk-4.0/4.0.0/printbackends
+
+  - name: openblas
+    sources:
+      - type: archive
+        url: https://github.com/OpenMathLib/OpenBLAS/archive/refs/tags/v0.3.28.tar.gz
+        sha256: f1003466ad074e9b0c8d421a204121100b0751c96fc6fcf3d1456bd12f8a00a1
+    buildsystem: simple
+    builddir: true
+    build-commands:
+      - export DYNAMIC_ARCH=1
+      - export USE_OPENMP=1
+      - export NUM_THREADS=56
+      - export TARGET=$(uname -m)
+      - export NO_SVE=1
+      - make CC=gcc FC=gfortran libs netlib shared -j1
+      - make PREFIX=/app install
+      - ln -s /app/lib/libopenblas.so /app/lib/libblas.so
+      - ln -s /app/lib/libopenblas.so /app/lib/liblapack.so
+
+  - name: bdw-gc
+    sources:
+      - type: git
+        url: https://github.com/ivmai/bdwgc
+        tag: v8.2.4
+    buildsystem: autotools
+    config-opts:
+      - --disable-debug
+      - --disable-docs
+      - --disable-dependency-tracking
+
+  - name: guile
+    sources:
+      - type: archive
+        url: https://ftp.gnu.org/gnu/guile/guile-2.2.7.tar.xz
+        sha256: cdf776ea5f29430b1258209630555beea6d2be5481f9da4d64986b077ff37504
+    buildsystem: autotools
+    config-opts:
+      - --with-libltdl-prefix=/app
+      - --with-libreadline-prefix=/app
+
+  - name: g-wrap
+    sources:
+      - type: archive
+        url: http://savannah.nongnu.org/download/g-wrap/g-wrap-1.9.15.tar.gz
+        sha256: 0ff6e2700e74b457323726f7c2551a466d8ba44339705f6792d7b533145c602a
+    buildsystem: autotools
+    config-opts:
+      - --disable-Werror
+
+  - name: guile-cairo
+    sources:
+      - type: archive
+        url: http://download.savannah.gnu.org/releases/guile-cairo/guile-cairo-1.10.0.tar.gz
+        sha256: ae3d7e0f0a8ae93d0a15c692514eb30f2478d407052064976fc59a291cccdd5c
+    buildsystem: autotools
+
+  - name: guile-lib
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://download.savannah.nongnu.org/releases/guile-lib/guile-lib-0.2.7.tar.gz
+        sha256: e4ef3b845f121882c7c0cf04f81a1cb8fd360c6f64b56b868de5546214f904de
+    config-opts:
+      - --with-guile-site=yes
+
+  - name: guile-gnome-platform
+    sources:
+      - type: archive
+        url: https://ftp.gnu.org/gnu/guile-gnome/guile-gnome-platform/guile-gnome-platform-2.16.5.tar.gz
+        sha256: 298d8c4f9b567bfe87beda18ed58d047c2e01b88c80895129de5466b921ccebe
+    buildsystem: autotools
+    build-options:
+      - CFLAGS: "-Wall"
```

Comment:
this proerty is unknow in `build-options`. Also how does `-Wall` impact the build?

Reply:
After setting `- CFLAGS: "-Wall"`, build got worked.

Reply:
I think we can lose `-Wall`

Reply:
https://github.com/flathub/flathub/pull/5690#discussion_r1795492713